interp/abbreviation.ml

```diff
@@ -73,11 +73,11 @@ let classify_abbreviation (local,_) =
 
 let inAbbreviation : Id.t -> (bool * abbreviation) -> obj =
   declare_named_object {(default_object "ABBREVIATION") with
-    cache_function = cache_abbreviation;
-    load_function = load_abbreviation;
-    open_function = simple_open open_abbreviation;
-    subst_function = subst_abbreviation;
-    classify_function = classify_abbreviation }
+    cache_function = cache_abbreviation
+    ; load_function = load_abbreviation
```

Comment:
This is ugly and also unnecessary diff

Reply:
Yes, it is an artifact of previous versions that did update this field tho.